### PR TITLE
add script to list all repos in an org for azure devops

### PR DESCRIPTION
**Pull request by gauravsaralMs**

### CODE REVIEW

File: devops/list-all-repos-in-org/list-all-repos-in-org.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# !/bin/sh
+
+organization="https://dev.azure.com/mseng"
+
+projects="$(az devops project list --org "${organization}" -o tsv --query [].name)"
```
Comment (sptramer) [inline devops/list-all-repos-in-org/list-all-repos-in-org.sh]: Requires the devops extension as a prereq. Make sure this is mentioned in a comment after the bang.

Comment (gauravsaralMs) [reply devops/list-all-repos-in-org/list-all-repos-in-org.sh]: added a comment, please take a look

Comment (gauravsaralMs) [reply devops/list-all-repos-in-org/list-all-repos-in-org.sh]: @sptramer : once this PR goes through, any idea where this script will be visible?
I mean are these scripts hosted/ exposed in some other portals/ sites as well?

Comment (sptramer) [reply devops/list-all-repos-in-org/list-all-repos-in-org.sh]: Only available on GitHub or with a specific markdown extension.

**Pull request merged**